ProjectSettings/ProjectVersion.txt

```diff
@@ -1 +1 @@
-m_EditorVersion: 5.6.2f1
+m_EditorVersion: 5.6.0f3
```

Comment:
Is there some way for me (without write access) to just accept your proposed changes?